packages/containers/src/List/List.connect.js

```diff
@@ -20,33 +15,25 @@ function getItems(state, config) {
 	return [];
 }
 
-export function mapDispatchToProps(dispatch, ownProps) {
-	const { name, id } = getContainerInfo(ownProps);
-	const props = getStateAccessors(dispatch, name, id, DEFAULT_STATE);
-	props.dispatch = dispatch;
-	return props;
-}
-
 export function mapStateToProps(state, ownProps) {
-	const { name, id } = getContainerInfo(ownProps);
-	const props = getStateProps(state, name, id);
+	const props = {};
 	const config = {
 		collectionId: ownProps.collectionId,
 		items: ownProps.items,
 	};
-
 	props.items = getItems(state, config);
```

Comment:
Since CMF manages collections, shouldn't the CMF connector resolve the collection items ?
* if `ownProps.collectionId`, CMF resolve it and inject the items (with a defined name, `collectionData` ?)
* in user's `mapStateToProps(state, ownProps, cmfProps)` (see comments in cmf connector), he can add his logics on `collectionData`, like filter

```
props.items = getItems(state, cmfProps.collectionData);
```